libswirl/gui/gui_renderer.cpp

```diff
@@ -581,6 +581,9 @@ struct GUIRenderer_impl : GUIRenderer {
     virtual void QueueEmulatorFrame(std::function<bool()> cb) {
         callback_mutex.Lock();
         frameDone = false;
+        #if FEAT_TA == TA_LLE
```

Comment:
These are here because slightly different timing / automagic frameskip between LLE and HLE